app/models/katello/docker_manifest.rb

```diff
@@ -9,6 +9,8 @@ class DockerManifest < Katello::Model
     CONTENT_TYPE = Pulp::DockerManifest::CONTENT_TYPE
     scoped_search :on => :name, :complete_value => true
     scoped_search :relation => :docker_tags, :on => :name, :rename => :tag, :complete_value => true, :only_explicit => true
+    scoped_search :on => :digest, :rename => :digest, :complete_value => true, :only_explicit => true
+    scoped_search :on => :schema_version, :rename => :schema_version, :complete_value => true, :only_explicit => true
```

Comment:
Both of these new searches are using `only_explicit: true` and I point that out because I think you'll have an upcoming PR which removes the name field from DockerManifest which means that all of the scoped search fields are `only_explicit: true`.

My question is: after removal of the name field, which, if any field should become `only_explicit: false` and should we make it that way at this time?

Reply:
Hmm good point I can see search by tag name being an automatic thing. That being said I would like to keep it the way it is in this PR because downstream will want what is here.

Reply:
That makes sense to me, thanks